sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImpl.java

```diff
@@ -806,6 +811,19 @@ public void deleteDataset(String projectId, String datasetId)
                         try {
                           return insert.execute().getInsertErrors();
                         } catch (IOException e) {
+                          GoogleJsonError.ErrorInfo errorInfo = getErrorInfo(e);
+                          if (errorInfo == null) {
+                            throw e;
+                          }
+                          /**
+                           * TODO:Check for quotaExceededError The check will be replaced by
```

Comment:
TODO(BEAM-10584): Check for QUOTA_EXCEEDED error will be replaced by

Reply:
Jira issue added